flake.nix

```diff
@@ -120,6 +120,7 @@
 
           preCheck = ''
             export RUST_BACKTRACE=1
+            export TEST_GIT_PATH=/usr/bin/git
```

Comment:
Said that in discord, but remove this and add git to `nativeBuildInputs` above - this shoud make git be there in PATH for nix tests, you can see how for similar reasons we added things like gnupg and openssh there